NineChronicles.Standalone/ActionEvaluationPublisher.cs

```diff
@@ -149,5 +163,58 @@ public override async Task StopAsync(CancellationToken cancellationToken)
             await _client?.DisposeAsync();
             await base.StopAsync(cancellationToken);
         }
+
+        private bool ContainsAddressToBroadcast(ActionBase.ActionEvaluation<ActionBase> ev)
+        {
+            if (_addressesToSubscribe is null ||
+                !_addressesToSubscribe.Any())
+            {
+                return false;
+            }
+            
+            var updatedAddresses = ev.OutputStates.UpdatedAddresses;
+            for (var i = _addressesToSubscribe.Count - 1; i >= 0; i--)
+            {
+                var address = _addressesToSubscribe[i];
+                if (updatedAddresses.Contains(address))
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        private void ResetAddressesToSubscribe(ActionBase.ActionEvaluation<ActionBase> ev)
+        {
+            if (!(ev.Action is CreateAvatar _) &&
+                !(ev.Action is CreateAvatar2 _))
+            {
+                return;
+            }
```

Comment:
What is the reason for excluding CreateAvatar?

Reply:
@ipdae This line is including `CreateAvatar` and `CreateAvatar2` action. When if not both and return.